pkg/connector/service.go

```diff
@@ -29,6 +29,11 @@ import (
 
 var idRegex = regexp.MustCompile(`^[A-Za-z0-9-_:.]*$`)
 
+const (
+	idLengthLimit   = 128
```

Comment:
@lovromazgon can we double the connector limits on the basis that `enrich` will add the pipeline name/id to the connector name. so probably 256+1?

Reply:
@lyuboxa is that a platform specific concern?

Reply:
@simonl2002 no, part of the pipeline files import